src/Bridge/Twig/Extension/EnumExtension.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Greg0ire\Enum\Bridge\Twig\Extension;
+
+/**
+ * @author Sullivan Senechal <soullivaneuh@gmail.com>
+ */
+final class EnumExtension extends \Twig_Extension
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getFunctions()
+    {
+        return [
+            new \Twig_SimpleFunction('enum_label', [$this, 'label']),
+        ];
+    }
+
+    /**
```

Comment:
Oh, I was thinking the reverse. An example as a filter:

```twig
{{ member.accessLevel | enum_label('AppBundle\\Enum\\OrganizationAccessLevel') | trans({}, 'global') }}
```

As we discussed, this is a "transformation" of a value onto a label...